server/chat-plugins/room-events.js

```diff
@@ -55,6 +55,7 @@ exports.commands = {
 				eventName: eventName,
 				date: date,
 				desc: desc,
+				timeRemaining: new Date(date).getTime() - new Date().getTime(),
```

Comment:
Gotta make sure that ``date`` is a valid parseable JS date.

Reply:
It's checked if `timeRemaining` is a false-y value in L25, should I move it to there?

Reply:
Uh........ Yeah..... ``timeRemaning`` shouldn't be stored, but validated and calculated on-demand.

Reply:
Done.